src/index.js

```diff
@@ -192,15 +191,13 @@ export const makeScrollToAction = scrollToFunc => {
         typeof current === "string" ? { element: current } : current
       );
     };
-    node.addEventListener("click", handle);
-    node.addEventListener("touchstart", handle);
+    _.addListeners(node, ["click", "touchstart"], handle);
```

Comment:
Just thought that having these be more generic and reusable was better than only using them in one place.